### ISSUE TITLE
Scratch activecode only available on some pages

### ISSUE THREAD

**Issue by tc-kirby** [RunestoneInteractive/rs#581]
I am finding that the 'scratch activecode' feature (pencil icon in the top right) is really helpful, but am only able to see it in the navigation from some pages e.g. Peer Instruction (Student) but not others e.g. Assignments.

Any chance it could be made available on all (or more of) the pages available from the 'person' dropdown?

Thanks.

**Comment by bnmnetp** [comment]
Indeed, it should be available on every page of every book.  But we need to get some metadata to the database to make this work on all of the instructor/assignment pages.  There is ongoing work to make this happen eventually.
